MANIFEST.in

```diff
@@ -2,20 +2,21 @@
 include CHANGES COPYING COPYING.LESSER
 
 # Files from setup.py package_data that are not automatically added to source distributions
-recursive-include lib/iris/tests/results *.cml *.cdl *.txt *.xml *.json
+recursive-exclude lib/iris/tests *
+recursive-exclude lib/iris/fileformats/_pyke_rules/compiled_krb *
 recursive-include lib/iris/etc *
-include lib/iris/fileformats/_pyke_rules/*.k?b
-include lib/iris/tests/stock*.npz
+include lib/iris/fileformats/_pyke_rules/*.krb
 
-include requirements/*.txt
+recursive-include requirements/ci *
 
 # File required to build docs
 recursive-include docs Makefile *.js *.png *.py *.rst
-prune docs/build
+prune docs/src/_build
```

Comment:
@tkknight Are there any other docs directories that shouldn't be packaged other than `docs/src/_build`?

Reply:
Perhaps these two:

* `docs/src/generated`.  This is for the API and gallery generated .rst files etc.
* `docs/gallery_tests`.  Tests for the gallery.

Reply:
@tkknight Done